internal/provider_test/suite_test.go

```diff
@@ -271,16 +271,34 @@ type TfHelper struct {
 // using `go test` (instrumentation of the provider binary does not seem to work
 // when Terraform is invoking it).
 func (tf *TfHelper) SetReattachConfig(config plugin.ReattachConfig) error {
-	encoded, err := json.Marshal(map[string]any{"registry.terraform.io/nuodb/nuodbaas": config})
+	encoded, err := json.Marshal(map[string]any{
+		"registry.terraform.io/nuodb/nuodbaas": config,
+		"registry.opentofu.org/nuodb/nuodbaas": config,
```

Comment:
This is the only change that was needed besides the plumbing to invoke `tofu` instead of `terraform`. Without this change `tofu show -json` fails because it cannot locate the schemas for the NuoDBaaS provider.

Reply:
Does this mean that `tofu show` depends on the publicly pushed version of the plugin?